### PR TITLE
Use selectors extracted during detection

### PR DESCRIPTION
**Pull request by atopper**
Show and save any selectors extracted during detection.
_NOTE - selector on any individual 'box' selected is not guaranteed (yet?)._

image

Example of saved JSON:
```
 
{
  color: "rgba(255, 255, 0, 0.4)"
  domClasses: "col-sm-3.col-md-3.sppb-in-article"
  domId: "sp-right"
  id: "box-id-8945113"
  layout: {numCols: 0, numRows: 0}
  mapping: "exclude"
  xpath: "/html[1]/body[1]/div[1]/div[1]/section[4]/div[1]/div[1]/div[2]"
}
```

### CODE REVIEW

File: js/import/import.ui.js [hunk 1]
```diff
@@ -591,7 +644,14 @@ const detectSections = async (src, frame) => {
...
       if (!mappingData.find((m) => m.id === section.id)) {
-        mappingData.push(section);
+        mappingData.push({
+          id: section.id,
+          domId: section.domId,
+          domClasses: section.domClasses,
+          xpath: section.xpath,
+          layout: section.layout,
+          color: section.color,
+        });
```
Comment (atopper) [inline js/import/import.ui.js:654]: This is the selection from the 'box' to maintain and save to JSON.  I kept the `id` and `classes` separated for now - felt more future-proof - but might want to collapse at some point.

**Review by arumsey — APPROVED** [review]
LGTM!

**Comment by arumsey** [comment]
image

If section mapping is targeting the non-developer/less technical user would it make sense to simplify this table?

- Color heading could be left out
- Change Id/Classes to simply `Selector`?
- Is layout useful? I don't know what it means.

**Comment by atopper** [comment]
```
Color heading could be left out
Change Id/Classes to simply Selector?
Is layout useful? I don't know what it means.
```

- Color matches the 'box' in the preview's color.  It makes it easier to spot it.
- Yes - `Selector` might be better.
- I've been playing with `Layout` to identify things like columns, and could use it for auto-selecting the mapping.  0x0 isn't helpful for sure, except to give you a hint that it is a simple component, but "N/A" might be more meaningful.

**Pull request merged**